### PR TITLE
ADM-331 [stub] Add mock data for commit data

### PR DESCRIPTION
**Pull request by liuxueming**
## Summary

Add mock data for commit data

## Before

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## After

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## Note

_Null_

### CODE REVIEW

**Review by guzhongren — APPROVED** [review]

File: stubs/backend/github/github-stubs.yaml [hunk 1]
```diff
@@ -53,3 +53,15 @@
...
+# Example: repos/xxx/xxx/commits/commit-sha
+- request:
+    method: GET
+    url: /repos/([\w.-]+\/[\w.-]+)/commits/(\b[0-9a-z]+\b)
+
+  response:
+    status: 200
+    headers:
+      content-type: application/json
+    file: ./backend/github/jsons/github.user.repos.commit.json
```
Comment (bhan-tw) [inline stubs/backend/github/github-stubs.yaml:67]: Do we need to rename the json file with `commit-sha`? 
And why we add `user` in the json file name? There is no keyword `user` in the request url.

Comment (liuxueming) [reply stubs/backend/github/github-stubs.yaml:67]: OK, will rename it.
Adding a `user` is a reference to other similar URLs, and we will discuss it with you offline

File: stubs/backend/github/github-stubs.yaml [hunk 2]
```diff
@@ -52,4 +52,16 @@
...
+# Example: repos/xxx/xxx/commits/commit-sha
+- request:
+    method: GET
+    url: /repos/([\w.-]+\/[\w.-]+)/commits/(\b[0-9a-z]+\b)
+
+  response:
+    status: 200
+    headers:
+      content-type: application/json
+    file: ./backend/github/jsons/github.repos.commit-sha.json
```
Comment (bhan-tw) [inline stubs/backend/github/github-stubs.yaml:67]: Should be ./backend/github/jsons/github.repos.fb1b445a113ef2d1ed465732a5dd02c2e07ca1c4.json
Incase we will add more commit as mock data.

**Review by bhan-tw — APPROVED** [review]
LGTM

**Pull request merged**